README.md

```diff
@@ -47,11 +47,11 @@ flowchart LR
 
 For more information about the Sindri platform, please check out [sindri.app](https://sindri.app/).
 The next section ([Getting Started](#getting-started)) shows how to launch a local Scroll SDK devote with provers running on Sindri.
-For production deployments, please consult the official [Scroll SDK documentation](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/production-deployment/).
+For production deployments, please consult the official [Scroll SDK documentation](https://docs.scroll.io/en/sdk/guides/production-deployment/).
```

Comment:
Um, this link is not provided in the sidebar of their own documentation, is this something we are okay with linking to?

Reply:
Good catch. It is featured in the sidebar in the old temporary domain but not here.  Presuming that is an intentional choice, I suppose we should just link the general docs and revisit this after that page is debuted. 
```suggestion
For production deployments, please consult the official [Scroll SDK documentation](https://docs.scroll.io/en/sdk/).
```